pkg/chains/formats/intotoite6/util/utils.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
Let's avoid the `util` name if possible, unless these are all just test helper funcs - https://go.dev/blog/package-names

If they are test helper funcs, put them into `pkg/internal/util` so it's not exposed as part of the library.

Reply:
Also for things like *RunFromFile that I'm pretty sure are only used in tests, you can enforce they are test only by having clients pass in a `t.Testing`, which also lets you t.Fatal if it fails and consolidates an error check that needs to be done in each test.

Reply:
Added them to the internal package. Although I left out the `t.Testing` parameter since this are used in the `init` functions to load the files once for tests.